README.rst

```diff
@@ -79,16 +79,19 @@ Project Status
 
 |Documentation Status| |Gitmate|
 
-Getting Involved
-----------------
+Newcomers Guide and Getting Involved
+------------------------------------
 
-If you want to contribute to coala, please take a look at the `Getting
-Involved Information
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__.
+coala is newcomer friendly and you will find a documentation
```

Comment:
a documentation? Sounds cute, you usually say `you will find documentation` or `you fill find a documentation page`. No really IMO this whole paragraph shouldn't be here, @Adrianzatreanu what was the reason for you to want to include it? It holds no additional info, everything important is covered below.